Resources/Prototypes/Entities/Structures/Walls/walls.yml

```
@@ -1106,13 +1106,14 @@
   parent: BaseWall
   id: WallWood
   name: wood wall
+  description: A wooden wall. Built by adding wood to a barricade.
```

There's not much point in putting the name of the item as the description. 

Additionally, we want to avoid putting anything related to how the construction as this will show when examined, where it doesn't need to be. If they need to know how to make it, they can find it from the construction menu.

Here's my suggestion (you don't need to use it), as a play on the whole wall keeping the graytide out.

```suggestion
  description: The traditional graytide defense.
```